Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1078,33 +1132,6 @@ async def coro():
         res = loop.run_until_complete(asyncio.wait_for(coro(), timeout=None))
         self.assertEqual(res, 'done')
 
-    def test_wait_for_with_global_loop(self):
```

Comment:
It is virtually a duplicate of the above test. Initially they were different by passing the loop argument or not, now it is removed